modules/machineset-customer-managed-encryption-azure.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-gcp.adoc
+
+[id="machineset-enabling-customer-managed-encryption-azure_{context}"]
+= Enabling customer-managed encryption keys for a machine set
+
+Azure allows users to supply an encryption key to encrypt data on managed disks at rest. You can enable server-side encryption with customer-managed keys by using the Machine API.
```

Comment:
⚠️ ISG prefers user-centered language over "allows" in cases like this. E.g., "You can supply an encryption key to Azure to encrypt data on managed disks at rest."